src/Bridges/Constraint/function_conversion.jl

```diff
@@ -16,7 +16,8 @@ function MOI.get(
         return MOI.get(model, attr, bridge.constraint)
     else
         throw(
-            ArgumentError(
+            MOI.UnsupportedAttribute(
+                attr,
                 "Bridge of type `$(typeof(bridge))` does not support accessing the attribute `$attr` because `MOIB.Constraint.invariant_under_function_conversion($attr)` returns `false`.",
```

Comment:
```suggestion
                "Bridge of type `$(typeof(bridge))` does not support getting the attribute `$attr` because `MOIB.Constraint.invariant_under_function_conversion($attr)` returns `false`.",
```